components/resource_control/src/service.rs

```diff
@@ -0,0 +1,266 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::{sync::Arc, time::Duration};
+
+use futures::StreamExt;
+use kvproto::{pdpb::ItemKind, resource_manager::ResourceGroup};
+use pd_client::{PdClient, Result, RpcClient};
+use tikv_util::{box_err, error};
+
+use crate::ResourceGroupManager;
+
+pub const CONFIG_PATH: &str = "resource_group/settings";
```

Comment:
Shall we declare this const path in the proto so both pd and tikv can depend on the same const. @nolouch what do you think

Reply:
After discussing, we think better maintain this const in client library. So let's move this into component pd_client instead.

Reply:
get it, Done.